### PR TITLE
Concise and readable projectConfig modifications

### PR DESCRIPTION
**Pull request by otterlistic**
Nested if condition was removed and repetition of projectConfig[key] has been avoided.
It's possible to mitigate the usage of forEach and filter on line 11 and 12 using reduce() which will be the next PR

### CODE REVIEW

File: src/tools/writeConfig.js [hunk 1]
```diff
@@ -15,16 +15,19 @@ const writeConfig = () => {
...
-            }
-          } else {
-            // Otherwise, set it
-            projectConfig[key] = file[key];
-          }
+         let configValue = projectConfig[key];
+         const fileValue = file[key];
+          if (configValue) {
+            
+            Array.isArray(configValue) && Array.isArray(fileValue))
```
Comment (sthielen) [inline src/tools/writeConfig.js]: I am getting a parser error here -- possibly requires some babel modifications to use this style of assignment?

File: src/tools/writeConfig.js [hunk 2]
```diff
@@ -15,16 +15,19 @@ const writeConfig = () => {
...
-            if (Array.isArray(projectConfig[key]) && Array.isArray(file[key])) {
-              projectConfig[key] = projectConfig[key].concat(file[key]);
-            } else {
-              projectConfig[key] = Object.assign(projectConfig[key], file[key]);
-            }
-          } else {
-            // Otherwise, set it
-            projectConfig[key] = file[key];
-          }
+         let configValue = projectConfig[key];
```
Comment (sthielen) [inline src/tools/writeConfig.js]: Can you check your editorconfig/eslint settings here? Should match indentation style for the rest of the project

**Review by sthielen — CHANGES_REQUESTED** [review]
Hey! Thanks for putting this together. Added some notes! If it's not too much trouble, want to add the updates using `reduce` to this PR too? I definitely agree that that is the right way forward

**Comment by otterlistic** [comment]
The changes have been made and reduce() has also been included

**Comment by sthielen** [comment]
Sweet! LGTM :)

**Pull request merged**